pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -241,6 +241,36 @@ func Convert_v1_Watchdog_To_api_Watchdog(source *v1.Watchdog, watchdog *Watchdog
 	return fmt.Errorf("watchdog %s can't be mapped, no watchdog type specified", source.Name)
 }
 
+func Convert_v1_Rng_To_api_Rng(source *v1.Rng, rng *Rng, _ *ConverterContext) error {
+
+	// default rng model for KVM/QEMU virtualization
+	rng.Model = "virtio"
+
+	// default backend model, random for /dev/random and /dev/urandom
+	rng.Backend = &RngBackend{
+		Model: "random",
+	}
+
+	// the source used to get the entropy
+	if source.Source == "" {
```

Comment:
it must be under validation webhook or we need to set it to some default

Reply:
I have provided the default in the `converter.go` since other defaults are also present here. 

As for the validation, it does not make any sense to validate the `source` field, it can be any valid device/path. The rates are set as `uint32` and the default validator handle negative case.

Reply:
does libvirt can work with any provided path?